spec/predicates/vuln.md

```diff
@@ -64,41 +64,39 @@ The `predicate` contains a JSON-encoded data with the following fields:
 > > > The version of the Vulnerability DB.
 
-**scanner.db.lastUpdate, required** string (timestamp)
+**scanner.db.lastUpdate, optional** string (timestamp)
 
 > > > The timestamp of when the vulnerability DB was updated last time.
 
-**scanner.result, required** list
+**scanner.result, required** object list
 
 > > The result contains a list of vulnerabilities. Note that an empty list means the **scanner** found no vulnerabilities.
 > > This is the most important part of this field because it'll store the scan result as a whole. So, people might want
 > > to use this field to take decisions based on them by making use of Policy Engines tooling whether allow or deny these images.
+> > Each object defines information about each one of the vulnerabilities found by the scanner.
 
-**scanner.result.[*].vulnerability, optional** object
 
-> > > The vulnerability object defines information about each one of the vulnerabilities found by the scanner.
+**scanner.result.[*].id, required** string
 
-**scanner.result.[*].vulnerability.id, required** string
+> > > This is the identifier of the vulnerability, e.g. [GHSA-fxph-q3j8-mv87](https://github.com/advisories/GHSA-fxph-q3j8-mv87) whose CVE id is [CVE-2017-5645](https://nvd.nist.gov/vuln/detail/CVE-2017-5645).
 
-> > > > This is the identifier of the vulnerability, e.g. [GHSA-fxph-q3j8-mv87](https://github.com/advisories/GHSA-fxph-q3j8-mv87) whose CVE id is [CVE-2017-5645](https://nvd.nist.gov/vuln/detail/CVE-2017-5645).
+**scanner.result.[*].severity, required** object list
 
-**scanner.result.[*].vulnerability.severity, required** object
+> > > The severity contains a list to describe the severity of a vulnerability using one or more quantitative scoring method.
 
-> > > > The severity contains a list to describe the severity of a vulnerability using one or more quantitative scoring method.
+**scanner.result.[*].severity.[*].method, required** string
 
-**scanner.result.[*].vulnerability.severity.method, required** string
+> > > > The method describes the quantitative method used to calculate the associated severity score such as nvd, cvss and others.
 
-> > > > > The method describes the quantitative method used to calculate the associated severity score such as nvd, cvss and others.
+**scanner.result.[*].severity.[*].score, required** string
 
-**scanner.result.[*].vulnerability.severity.score, required** string
+> > > > This is a string representing the severity score based on the selected method.
 
-> > > > > This is a string representing the severity score based on the selected method.
+**scanner.result.[*].annotations, optional** object list
 
-**scanner.result.[*].vulnerability.annotations, optional** list
+> > > This is a list of key/value pairs where scanners can add additional custom information.
 
-> > > > > This is a list of key/value pairs where scanners can add additional custom information.
-
-**metadata.scanStartedOn, required** Timestamp
+**metadata.scanStartedOn, optional** Timestamp
```

Comment:
I have the same thoughts as above. This must be required to ensure all the details to reuse these attestations in the future.

Reply:
i think i run into the same problem where scanners today don't provide that information as well? is there a particular use of the scan start time you're thinking of?

Reply:
This information is relevant for many scanners such as Grype, depending on the date of last update the report could be discarded.